test/integration/glob.spec.js

```diff
@@ -28,7 +28,7 @@ describe('globbing', function() {
           expect(
             results.stderr,
             'to contain',
-            'cannot find any files matching pattern "./*-none.js"'
+            'Error: Cannot find any files matching pattern "./*-none.js"'
```

Comment:
test pattern (filename) should not be part of check for maintainability.

Reply:
Wasn't if on purpose, I thought there was nothing outstanding so I merged it. 
Have made a new PR continuing the work.